block/polka.go

```diff
@@ -22,7 +29,10 @@ func NewPreVote(block *Block, round Round, height Height) PreVote {
 	}
 }
 
+// Sign is indented to be the way you sign your PreVote with your
+// private key
 func (preVote PreVote) Sign(signer sig.Signer) (SignedPreVote, error) {
+	//FIXME: this does not guarantee the contents of the block
```

Comment:
Yes it does. Because the block header is inside the String method of the PreVote.